README.md

```diff
@@ -2,25 +2,16 @@
 
 This is a tree-sitter parser grammar for the [pug](https://github.com/pugjs/pug) language.
 
-For now this is intended to be used **for a vue+pug** setup, as the control flow operations aren't recognized by the parser (since they are not used in conjunction with vue).
+This is intended to be used along side Angular, but my aim is to eventually support all pug features.
```

Comment:
Can you word this differently? It's not for either Vue or Angular only now :)

Reply:
Thanks @zealot128 I've update the  README to be more inclusive :+1: